pdfminer/pdfinterp.py

```diff
@@ -204,6 +212,14 @@ def get_font(self, objid, spec):
                 if settings.STRICT:
                     raise PDFFontError('Invalid Font spec: %r' % spec)
                 font = PDFType1Font(self, spec)  # this is so wrong!
+            if objid and self.bbox_per_font_and_glyph is not None:
+                # If available, set glyph boxes for each font
+                if font.fontname in self.bbox_per_font_and_glyph:
+                    font.glyph_bounding_box = self.bbox_per_font_and_glyph[font.fontname]
+            if objid and self.bbox_per_font is not None:
+                # If available, set a minimal bounding box that bounds all glyph boxes for each font
```

Comment:
To make sure I'm understanding, it's minimal across all glyphs in the font, but not necessarily for any particular one right?